models/biocro/R/model2netcdf.BIOCRO.R

```diff
@@ -71,14 +72,19 @@ model2netcdf.BIOCRO <- function(result, genus = NULL, outdir, lat = -9999, lon =
     
     k <- udunits2::ud.convert(1, "Mg/ha", "kg/m2") / c2biomass
```

Comment:
Not following why k is divided by c2biomass since most (possibly all) of the variables multiplied by k are supposed to be reported on a carbon basis, not a biomass basis. If BioCro internal pools are on a biomass basis, you should multiply by c2biomass. If they're already on a carbon basis, you don't need this.